src/controller/python/chip/clusters/ClusterObjects.py

```diff
@@ -228,6 +234,16 @@ class Cluster(ClusterObject):
     especially the TLV decoding logic. Also ThreadNetworkDiagnostics has an attribute with the same name so we
     picked data_version as its name.
     '''
+
+    def __init_subclass__(cls, *args, **kwargs) -> None:
+        """Register a subclass."""
+        super().__init_subclass__(*args, **kwargs)
+        # register this cluster in the ALL_CLUSTERS dict for quick lookups
+        try:
+            ALL_CLUSTERS[cls.id] = cls
+        except NotImplementedError:
+            pass # we can safely ignore this
```

Comment:
When would there be a NotImplementedError here at all?  When reading cls.id?

Reply:
adjusted the comment.